qubespolicy/__init__.py

```diff
@@ -121,11 +125,11 @@ def __init__(self, line, filename=None, lineno=None):
         self.filename = filename
 
         try:
-            self.source, self.target, self.full_action = line.split()
+            self.source, self.target, self.full_action = line.split(maxsplit=2)
         except ValueError:
             raise PolicySyntaxError(filename, lineno, 'wrong number of fields')
 
-        (action, *params) = self.full_action.split(',')
+        (action, *params) = self.full_action.replace(' ', '').split(',')
```

Comment:
arguments to the action are still separated by coma(s). The intention here is to tolerate spaces, as many would add after coma